src/views/Wrap/Wrap.tsx

```diff
@@ -217,155 +217,155 @@ const Wrap: React.FC = () => {
-            <Grid container direction="column" spacing={2}>
-              <Grid item>
-                <div className="card-header">
-                  <Typography variant="h5">Wrap / Unwrap</Typography>
-                  <Link
-                    className="migrate-sohm-button"
-                    style={{ textDecoration: "none" }}
-                    href={
-                      assetTo === "wsOHM"
-                        ? "https://docs.olympusdao.finance/main/contracts/tokens#wsohm"
-                        : "https://docs.olympusdao.finance/main/contracts/tokens#gohm"
-                    }
-                    aria-label="wsohm-wut"
-                    target="_blank"
-                  >
-                    <Typography>gOHM</Typography>{" "}
-                    <SvgIcon component={ArrowUp} color="primary" style={{ marginLeft: "5px", width: ".8em" }} />
-                  </Link>
-                </div>
-              </Grid>
-              <Grid item>
-                <MetricCollection>
-                  <Metric
-                    label={`sOHM ${t`Price`}`}
-                    metric={formatCurrency(sOhmPrice, 2)}
-                    isLoading={sOhmPrice ? false : true}
-                  />
-                  <Metric
-                    label={t`Current Index`}
-                    metric={trim(currentIndex, 1)}
-                    isLoading={currentIndex ? false : true}
-                  />
-                  <Metric
-                    label={`gOHM ${t`Price`}`}
-                    metric={formatCurrency(gOhmPrice, 2)}
-                    isLoading={gOhmPrice ? false : true}
-                    tooltip={`gOHM = sOHM * index\n\nThe price of gOHM is equal to the price of sOHM multiplied by the current index`}
-                  />
-                </MetricCollection>
-              </Grid>
-              <div className="staking-area">
-                {!address ? (
-                  <div className="stake-wallet-notification">
-                    <div className="wallet-menu" id="wallet-menu">
-                      {modalButton}
-                    </div>
-                    <Typography variant="h6">Connect your wallet</Typography>
+          <Paper
+            className="ohm-card"
+            headerText="Wrap / Unwrap"
+            topRight={
+              <Link
+                className="migrate-sohm-button"
+                style={{ textDecoration: "none" }}
+                href={
+                  assetTo === "wsOHM"
+                    ? "https://docs.olympusdao.finance/main/contracts/tokens#wsohm"
+                    : "https://docs.olympusdao.finance/main/contracts/tokens#gohm"
+                }
+                aria-label="wsohm-wut"
+                target="_blank"
+              >
+                <Typography>gOHM</Typography>{" "}
+                <SvgIcon component={ArrowUp} color="primary" style={{ marginLeft: "5px", width: ".8em" }} />
+              </Link>
```

Comment:
Or instead of all of this you could do: 
```suggestion
              <TextButton
                style={{ height: "inherit", margin: "0", padding: "0" }}
                href={
                  assetTo === "wsOHM"
                    ? "https://docs.olympusdao.finance/main/contracts/tokens#wsohm"
                    : "https://docs.olympusdao.finance/main/contracts/tokens#gohm"
                }
              >
                gOHM
              </TextButton>
```
to avoid importing Icon or Link alltogether. Totally optional, just a suggestion

Reply:
cool, i think this suggestion looks much cleaner. done.

Reply:
Icon is too big for this button/link in the library so I am using the icon component from the library instead.